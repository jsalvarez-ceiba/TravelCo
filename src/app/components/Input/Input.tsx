import React, { FC, InputHTMLAttributes } from 'react';

interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
    value:string | number;
    name:string;
    type:string;
}

const Input: FC<InputProps> = ( {value, name, type, ...rest} : InputProps ) => {
    return (
        <input value={value} name={name} {...rest} type={type} className="form-control" />
    );
};

export default Input;
