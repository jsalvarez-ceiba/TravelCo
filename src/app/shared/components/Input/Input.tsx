import React, { FC, InputHTMLAttributes } from 'react';

interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
    value:string | number;
    name:string;
    type:string;
}

const Input: FC<InputProps> = ( {value, name, type, max,...rest} : InputProps ) => {
    return (
        <input value={value} name={name} {...rest} type={type} max={max} className="form-control" />
    );
};

export default Input;
