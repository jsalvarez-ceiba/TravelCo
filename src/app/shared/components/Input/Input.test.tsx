import React from 'react';
import renderer from 'react-test-renderer';
import Input from './Input';

describe('InputComponent Test', () => {
    const input = renderer.create(<Input />).toJSON();
    test('Input Render', () => {
        expect(input).toMatchSnapshot();
    });
});