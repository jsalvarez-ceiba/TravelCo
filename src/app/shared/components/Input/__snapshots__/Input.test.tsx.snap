// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputComponent Test Input Render 1`] = `
<input
  className="form-control"
/>
`;
